src/main/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporter.java

```diff
@@ -179,4 +190,12 @@ public void export(final OutputStream stream) throws IOException {
         writer.flush();
     }
 
+    private String buildCounterName(OperationType type) {
+        return ADMIN_EVENT_PREFIX + type.name();
```

Comment:
I don't think it is an issue in this specific code, but I would use `StringBuilder` here instead of the string concatenation

Reply:
you're right @ziccardi 

but I will leave it :) too minor